pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -14,6 +14,9 @@
 from rest_framework.generics import get_object_or_404
 from rest_framework.schemas import AutoSchema
 
+NAME_FILTER_OPTIONS = ['exact', 'in']
```

Comment:
Wow, nice catch, "in" is indeed valid.
https://docs.djangoproject.com/en/dev/ref/models/querysets/#in

This class threw me off:
http://django-filter.readthedocs.io/en/master/ref/filters.html#baseinfilter

This DRYs things up quite a bit.

Reply:
It surprised me to.  It works *automagically*

Reply:
Aha! I did my initial work on filtering before this issue was filed, and did exactly what the reporter did.